config/config.go

```diff
@@ -0,0 +1,115 @@
+package config
+
+import (
+	"fmt"
+	"time"
+)
+
+// Serialized defines the standard config for this library.
+type Serialized struct {
+	Report      bool   `json:"report"`
+	StoragePath string `json:"storagePath"`
+	Interval    string `json:"interval"`
+	Endpoint    string `json:"endpoint"`
+	PullReport  struct {
+		Endpoint     string `json:"endpoint"`
+		Delay        string `json:"delay"`
+		Timeout      string `json:"timeout"`
+		MinRetryTime string `json:"min_retry"`
+	} `json:"pull_report"`
```

Comment:
We don't need this. Maybe you wanted to make the clean up in the next step, but I would keep it simple as possible. Just my 2 cents. :-)

Reply:
Yeah. Makes sense. I removed the `Serialized` struct and the `ToConfiguration` function.